dev/documentation/en/utilities-hooks.md

```diff
@@ -109,9 +109,9 @@ The following hooks allow plugins to intervene in request handling and routing.
 
 ### Entities
 
-The following hooks allow plugins to intervene in how [Entities](./architecture-entities) are fetched, validated, added and edited. Use these hooks to modify the properties of an object.
+The following hooks allow plugins to intervene in how [Entities](/dev/documentation/en/architecture-entities) are fetched, validated, added and edited. Use these hooks to modify the properties of an object.
```

Comment:
Thanks @kaitlinnewson. Just so I'm aware, what's the thinking on why to use the full relative URL (`/`)? Using the `./` would mean the URL would work if the content is ported to any other language. But I may be missing something too. Happy to use either approach.

Reply:
No reason to use the full relative URL necessarily, just wanted to make it consistent with the formatting of other links. I believe we also use the full relative URLs throughout most of the docs hub.

Reply:
Ok sounds good. For reference, I usually use the `./` to link to pages at the same level. So linking between `/dev/documentation/en/architecture.md` and `/dev/documentation/en/frontend.md`. And then use `/...` to link to anything else. 

The thinking is that if a guide is moved a `./` link will automatically work, whereas a `/dev/...` won't. But I wouldn't want the distinction to be a source of confusion and what's clearest will be the easiest to maintain. So let's go with `/...` for links. :+1: